### PR TITLE
Manually move future migrations

### PR DESCRIPTION
**Pull request by joseph-flinn**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [X] Other
```

## Objective

Manually orchestrating our EDD processes: moving the future scripts (soon to be officially known as Finalization migrations).

## Code changes

* **util/Migrator/DbScripts/2023-09-11_00_2023-01-FutureMigration.sql:** Move future migration to be run this release
* **util/Migrator/DbScripts/2023-09-11_01_2023-02-FutureMigration.sql:** Move future migration to be run this release
* **src/Sql/dbo/*:** move the files from `src/Sql/dbo_future` to `src/Sql/dbo`

## Before you submit

- Please check for formatting errors (`dotnet format --verify-no-changes`) (required)
- If making database changes - make sure you also update Entity Framework queries and/or migrations
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

**Review by Hinton — CHANGES_REQUESTED** [review]
I believe we should also move `dbo_future` and overwrite existing files.

And the migrations most likely won't work since `User_Create` has new columns that don't seem to be reflected.

**Comment by Hinton** [comment]
@joseph-flinn run git blame and ping the authors to update the scripts.

**Comment by joseph-flinn** [comment]
@Hinton The `src/Sql/dbo_future` scripts have all been moved to their corresponding `src/Sql/dbo` directories

File: src/Sql/dbo/Tables/User.sql
Comment (Hinton) [inline src/Sql/dbo/Tables/User.sql:1]: @fedemkr this file seems to need updating.

**Review by Hinton — DISMISSED** [review]
Looks like everything is updated as expected

**Comment by joseph-flinn** [comment]
After discussing with QA, we are going to hold this PR for the October release.

**Comment by joseph-flinn** [comment]
@Hinton I need to request another review since https://github.com/bitwarden/server/pull/3192 changed the directory name

**Review by Hinton — APPROVED** [review]

**Pull request merged**
